homeassistant/components/telegram_bot/__init__.py

```diff
@@ -844,6 +869,28 @@ def send_location(self, latitude, longitude, target=None, **kwargs):
                 timeout=params[ATTR_TIMEOUT],
             )
 
+    def send_poll(self, question, options, target=None, **kwargs):
```

Comment:
define all args that have a default value set in the schema. Those will always be present.